cms/templates/library.html

```diff
@@ -29,8 +33,8 @@
             {
                 isUnitPage: false,
                 page_size: 10,
-                canEdit: ${"true" if can_edit else "false"},
-                showChildrenPreviews: ${'true' if show_children_previews else 'false'}
+                canEdit: ${can_edit | n, dump_js_escaped_json},
+                showChildrenPreviews: ${context_library.show_children_previews | n, dump_js_escaped_json}
```

Comment:
I know `show_children_previews` will always be `True` or `False`: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/library_root_xblock.py#L39-L44.

The thing that was throwing me off is that `context_library` is being passed to this template. So if we want this value, we need to evaluate `context_library.show_children_previews`. `show_children_previews` by itself doesn't refer to anything. Normally, we'd expect this to throw a `NameError`, but mako, by default, evaluates it to `False`.